### DISCUSSION TITLE
GitHub API: List repositories endpoint does not filter by type=private

### DISCUSSION THREAD

**Discussion by FernandoZX** [community/community#45581] [Apps, API and Webhooks]
### Select Topic Area

Show & Tell

### Body

After doing the identify & authorize user process using via github apps [identifying-and-authorizing-users-for-github-apps](https://docs.github.com/en/developers/apps/building-github-apps/identifying-and-authorizing-users-for-github-apps)

I was requesting on the endpoints for fetch org repos and my repos return an empty array following from the api documentation and enble all permision in my github Apps.

`https://api.github.com/orgs/my_org/repos?per_page=100&type=private` this is the endpoint for fetch org repos private
[list-organization-repositories](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-organization-repositories)

`https://api.github.com/user/repos?visibility=private`this is the endpoint for fetch my repos visibility private.
[list-repositories-for-the-authenticated-user](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-repositories-for-the-authenticated-user)

Could provide a screenshot for the proper steps of the implementations?
Could provide the proper API documentation?

Kind regards.

**Comment by jge162** [comment]
[Endpoint documentation](https://docs.github.com/en/rest/overview/endpoints-available-for-github-apps?apiVersion=2022-11-28) @FernandoZX 

please see post here also with solution #[39719](https://github.com/orgs/community/discussions/39719)

**Reply by jge162** [reply]
if you are satisfied with answer please consider closing ticket or ask me something else. thank you so much.

**Reply by FernandoZX** [reply]
i followed the solution and still having the same issue
